src/google/protobuf/json/internal/parser.cc

```diff
@@ -7,12 +7,11 @@
 
 #include "google/protobuf/json/internal/parser.h"
 
-#include <cfloat>
```

Comment:
Similarly, header removals might *seem* like cleanups, but we have hidden internal code that could be using this.  So this might just end up making our merges messier